File: src/class/wallet.dart
class Wallet {
  String color;
  String mark;
  bool openStatus = false;

  Wallet(this.color, this.mark);

  String getColor() => color;

  String getMark() => mark;

  bool isOpen() => openStatus;

  void open() {
    openStatus = true;
  }

  void close() {
    openStatus = false;
  }
}

File: src/class/Wallet.js
export class Wallet {
  constructor(color, mark) {
    this.color = color;
    this.mark = mark;
    this.openStatus = false;
  }

  getColor = () => this.color;

  getMark = () => this.mark;

  isOpen = () => this.openStatus;

  open = () => {
    this.openStatus = true;
  };

  close = () => {
    this.openStatus = false;
  };
}
